documentation/modules/rn-2.6.adoc

```diff
@@ -0,0 +1,434 @@
+
+.Conversion pod does not use the transfer network
+
+When creating a migration plan using the `transferNetwork`, the `virt-v2v` pod uses the default pod network instead of the transfer network. link:https://issues.redhat.com/browse/MTV-835[(MTV-835)]
+
+.Creating a host secret requires validating the secret before creating the host
+
+When creating a host secret, you must validate the secret before creating the host. The procedure for creating a host secret is as follows:
+
+. Create a secret
+. Create a host
+. Update the host owner reference
+
+link:https://issues.redhat.com/browse/MTV-868[(MTV-868)]
+
+.Migrating VMs from RHV to {ocp-name} Virtualization does not work with an encrypted RBD
+
+Migrating a VM from RHV to {ocp-name} Virtualization fails during `qemu-img create`, returning a `Cannot grow device files` error. The pod sees a smaller file. link:https://github.com/kubev2v/forklift-console-plugin/issues/852[(MTV-852)]
+
+.RHV provider `ConnectionTestSucceeded` True response from wrong URL
+
+The `ConnectionTestSucceeded` process produces a `True` result even when the wrong URL is entered. link:https://issues.redhat.com/browse/MTV-740[(MTV-740)]
+
+.MTV Inventory does not show extended volume of VM disk in RHV
+
+The extended volume of a VM disk does not appear in the MTV Inventory for an RHV provider after it is already connected. The connected RHV provider does not update the volume of the VM disk. link:https://issues.redhat.com/browse/MTV-829[(MTV-829)]
+
+.`ForkliftController` status shows `Failure` after upgrading from {project-short} 2.4.2 to {project-short} 2.5.0
+
+After upgrading {project-short} from version 2.4.2 to 2.5.0, the `ForkliftController` returns a `Failure` status and an error message. link:https://issues.redhat.com/browse/MTV-702[(MTV-702)]
+// 
+// .Migration fails when a vSphere Datacenter is nested inside a folder
+// 
+// Migrating a vSphere Datacenter succeeds when it is directly under the `/vcenter`, but fails when it is stores inside a folder. The migration raises an error. link:([MTV-796])
+
+.Fail to migrate a VM with NVME disks from vSphere 
+
+When migrating a VM with NVME disks from vSphere, the migration process fails and the Web Console shows that the `Convert image to kubvirt` stage is 'running` but did not finish successfully. link:https://issues.redhat.com/browse/MTV-963[(MTV-963)]
+
+.Fail to migrate image-based VM from {osp} to `default` namespace
+
+The migration process fails when migrating an image-based VM from {osp} to the `default` namespace. The migration does not have a `RunAsUser` and `FSGroup`. link:https://issues.redhat.com/browse/MTV-964[(MTV-964)]
```

Comment:
I think the `The migration does not have a RunAsUser and FSGroup` part is too technical and not informative enough for users, how about something like:
```suggestion
The migration process fails when migrating an image-based VM from {osp} to the `default` namespace. The conversion of a virtual machine snapshot as part of the migration runs with non-root privileges and OpenShift does not automatically pick non-root user for a pod that starts on the `default` namespace. link:https://issues.redhat.com/browse/MTV-964[(MTV-964)]
```
@bennyz what do you think?

Reply:
@ahadas 
Changed according to your comment.
Also changed 'namespace' to project.